core/src/main/scala/org/scalarelational/model/ModularSupport.scala

```diff
@@ -11,19 +12,28 @@ import org.scalarelational.instruction._
  *
  * @author Matt Hicks <matt@outr.com>
  */
-trait ModularSupport extends SQLContainer with Listenable with DataTypes {
+trait ModularSupport extends SQLContainer {
+  case class Processor[T]() {
+    val functions = ArrayBuffer.empty[T => T]
+    def fire(value: T): T =
```

Comment:
I would like to rename this method. In MetaRx I'm using ``produce()`` for channels.